### PR TITLE
new post panel, remove rounded subheader for mobile

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

File: src/dotnet/UI.Blazor.App/Components/Author/AuthorCircleGroup.razor [hunk 1]
```diff
@@ -40,6 +61,7 @@
     [Parameter] public string Class { get; set; } = "";
     [Parameter] public int MaxIconCount { get; set; } = 5;
     [Parameter] public SquareSize Size { get; set; } = SquareSize.Size6;
+    [Parameter] public bool IsForTesting { get; set; }
```
Comment (frolyo) [inline src/dotnet/UI.Blazor.App/Components/Author/AuthorCircleGroup.razor]: It's a bad pattern for production code.

Comment (frolyo) [reply src/dotnet/UI.Blazor.App/Components/Author/AuthorCircleGroup.razor]: And I can't find where you use it.

File: src/dotnet/UI.Blazor.App/Components/ChatHeader.razor [hunk 2]
```diff
@@ -42,17 +42,14 @@
                             <ChatMemberCounter ChatContext="@ChatContext"/>
                         }
                     }
-                    @if (!ScreenSize.IsNarrow()) {
+                    @if (chat.Id.IsPeerChat(out _)) {
```
Comment (frolyo) [inline src/dotnet/UI.Blazor.App/Components/ChatHeader.razor:45]: there was property Kind

**Review by frolyo — CHANGES_REQUESTED** [review]

**Review by frolyo — APPROVED** [review]

File: src/dotnet/UI.Blazor.App/Components/Banners/Banners.razor [hunk 3]
```diff
@@ -5,10 +5,9 @@
 }
 
 <ReconnectBanner/>
-@* TODO: need to update (Andrey Y) *@
-@* @if (ScreenSize.IsNarrow()) { *@
-@*     <TalkingMembersBanner ChatContext="@ChatContext"/> *@
-@* } *@
+@if (ScreenSize.IsNarrow()) {
+     <TalkingMembersBanner ChatContext="@ChatContext"/>
```
Comment (alexyakunin) [inline src/dotnet/UI.Blazor.App/Components/Banners/Banners.razor:9]: Rename it to "ChatActivityBanner"

File: src/dotnet/UI.Blazor.App/Components/Banners/banners.css [hunk 4]
```diff
@@ -47,12 +47,11 @@ body.narrow .banner.reconnect-banner .banner-body > .c-main-text.desktop {
 }
 
 .talking-members-banner.banner {
```
Comment (alexyakunin) [inline src/dotnet/UI.Blazor.App/Components/Banners/banners.css:49]: And this as well.

File: src/dotnet/UI.Blazor.App/Components/TalkingMembersPanel/talking-members-panel.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    @apply flex-x items-center gap-x-2;
+    @apply h-14;
+}
+body.wide .talking-members-panel {
+    @apply h-10;
+    @apply pl-2 pr-1;
+    @apply rounded-lg;
+    @apply bg-03;
+}
+.talking-members-panel .silence-icon {
```
Comment (alexyakunin) [inline src/dotnet/UI.Blazor.App/Components/TalkingMembersPanel/talking-members-panel.css:11]: And here.

**Review by alexyakunin — CHANGES_REQUESTED** [review]

**Pull request merged**